apps/google-analytics-4/lambda/src/lib/verify-signed-request-middleware.ts

```diff
@@ -37,13 +39,13 @@ export const verifySignedRequestMiddleware = (
   next();
 };
 
-const makeCanonicalReq = (req: express.Request) => {
+const makeCanonicalReq = (req: express.Request): CanonicalRequest => {
   const requiredHeaders = requestSigningHeaders(req.headers);
 
-  return <CanonicalRequest>{
-    method: req.method,
+  return {
+    method: req.method as HttpMethod,
```

Comment:
Would it just except `string` instead of relying on the google api lib? (per my comment above)